File: Pick peaks.js
function pickPeaks(arr){  
  //  return {pos:[],peaks:[]}
  let pos = [], peaks = [];
  for (let i = 1; i < arr.length-1; i++)
  {
    if (arr[i] > arr[i-1])
    {
      for (let j = i + 1; j < arr.length; j++)
      {
        if (arr[j] > arr[i])
          break;
        else if (arr[j] < arr[i])
        {
          pos.push(i);
          peaks.push(arr[i]);
          break;
        }
      }
    }
  }
  return {pos: pos, peaks: peaks};
}

File: 4-kyu/Pick peaks.py
def pick_peaks(arr):
    #your code here
    pos = []
    peaks = []
    for i in xrange(1, len(arr)-1):
        if arr[i] > arr[i-1]:
            for j in xrange(i+1, len(arr)):
                if arr[j] > arr[i]:
                    break
                elif arr[j] < arr[i]:
                    pos += [i]
                    peaks += [arr[i]]
                    break
    return {'pos': pos, 'peaks': peaks}
